### DISCUSSION TITLE
Custom Front End

### DISCUSSION THREAD

**Discussion by paulpsky** [Budibase/budibase#1656] [Ideas]
Can Budibase be used to JUST build backend structure? allowing a custom (non budibase) front end HTML/CSS/ReactJS to be hooked up?

**Comment by shogunpurple** [comment]
Hi @paulpsky - it can, although the backend API right now is undocumented and not ready for real use cases. 

We will be implementing a very friendly backend API with documentation to match over the coming months, which will allow you to use budibase entirely as a backend for an existing frontend application.
